prow/config/config.go

```diff
@@ -147,6 +148,15 @@ type Plank struct {
 	PodPendingTimeout time.Duration `json:"-"`
 }
 
+// Gerrit is config for the gerrit controller.
+type Gerrit struct {
+	// TickInterval is how often we do a sync with binded gerrit instance
+	TickIntervalString string        `json:"tick_interval,omitempty"`
+	TickInterval       time.Duration `json:"-"`
+	// RateLimit defines how many changes to query per gerrit API call
+	RateLimit int `json:"ratelimit,omitempty"`
```

Comment:
Please mentions the default values in the GoDoc comments.

Reply:
done, and added default